assets/js/webflow/import.js

```diff
@@ -73,34 +73,13 @@ ga('create', '{{ GOOGLE_ANALYTICS_ID }}', 'auto');\
 ga('send', 'pageview');\
 ");
 
-$(".section--home-hero .container .layout-grid .layout-grid__col").prepend("{% for notice in site_notices %}\
-<div>\
-  {{ notice.content | safe }}\
-</div>\
-{% endfor %}\
-");
-
-$(".profile-notice").html("{% for notice in site_notices %}\
+$(".site-notice").html("{% for notice in site_notices %}\
```

Comment:
Minor but in future you can use a JavaScript literal to do multiilines rather than having to `\` at the end of each line.  e.g.
```
$(".site-notice").html(`{% for notice in site_notices %}
  <div class='container'>
    <div class='site-notice__text' style='color:white;margin-top:10px;'>{{ notice.content | safe }}</div>
  </div>
{% endfor %}
`);
```

Reply:
Changed this section to a JS literal, most of the import script uses slashes but will use literals from now on